apps/mobile/app.json

```diff
@@ -2,11 +2,14 @@
   "expo": {
     "name": "tequila-client",
```

Comment:
Btw I'm also thinking about this name. Why don't we use something like `margarita`?

Reply:
Say no more, maybe something like  `margarita-mobile`?

Reply:
Yes, it's fine. Pls update also url for Expo link in README.md.

Reply:
Done!